examples/custom_path_solvers/example.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2019 The TensorNetwork Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+import numpy as np
+import tensornetwork as tn
+from tensornetwork import contractors
+from tensornetwork.ncon_interface import ncon
```

Comment:
change this to 

`from tensornetwork import ncon`.

Also, please move this file to the `examples` directory at the root of the repository.

Reply:
I would suggest that `ncon_solver` could be added to the main `__init__.py` file as it is intended to be used in conjunction with `ncon`.